subspace_model/params.py

```diff
@@ -105,21 +106,32 @@ def DEFAULT_SLASH_FUNCTION(state: SubspaceModelState):
     transfer_holder_to_operator_per_day=0.01, # TODO
 
     # Environmental Parameters
-    avg_base_fee=5,
-    std_base_fee=15,
+    avg_base_fee=1,
```

Comment:
Note: as of now Base Fee have SSC/ComputeWeights unit. We're going to change that to Shannon per Compute Weights. This may affect that number

Reply:
It is 1 shannon in the protocol rn, but we want to change it to something better with guidance from your side